### PR TITLE
Announcing Scala.js 1.9.0.

### PR DESCRIPTION
**Pull request by sjrd**
To be merged later, once it is published.

### CODE REVIEW

File: _posts/news/2022-02-11-announcing-scalajs-1.9.0.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+Bug reports can be filed [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Release notes
+
+If upgrading from Scala.js 0.6.x, make sure to read [the release notes of Scala.js 1.0.0]({{ BASE_PATH }}/news/2020/02/25/announcing-scalajs-1.0.0/) first, as they contain a host of important information, including breaking changes.
+
+This is a **minor** release:
+
+* It is backward binary compatible with all earlier versions in the 1.x series: libraries compiled with 1.0.x through 1.8.x can be used with 1.9.0 without change.
+* It is *not* forward binary compatible with 1.8.x: libraries compiled with 1.9.0 cannot be used with 1.8.x or earlier.
```
Comment (sjrd) [inline _posts/news/2022-02-11-announcing-scalajs-1.9.0.md:36]: This is lie; it is actually forward binary compatible, for once. But IMO it's easier to communicate it this way not to create some confusion.

**Comment by sjrd** [comment]
Ping @gzm0 ?

File: _posts/news/2022-02-13-announcing-scalajs-1.9.0.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+This is a **minor** release:
+
+* It is backward binary compatible with all earlier versions in the 1.x series: libraries compiled with 1.0.x through 1.8.x can be used with 1.9.0 without change.
+* It is *not* forward binary compatible with 1.8.x: libraries compiled with 1.9.0 cannot be used with 1.8.x or earlier.
+* It is *not* entirely backward source compatible: it is not guaranteed that a codebase will compile *as is* when upgrading from 1.8.x (in particular in the presence of `-Xfatal-warnings`).
+
+As a reminder, libraries compiled with 0.6.x cannot be used with Scala.js 1.x; they must be republished with 1.x first.
+
+## Strict floats by default
```
Comment (gzm0) [inline _posts/news/2022-02-13-announcing-scalajs-1.9.0.md:42]: I think this should mention somewhere that strict floats is a linking time decision (and therefore applies to the whole program).

File: _posts/news/2022-02-13-announcing-scalajs-1.9.0.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+Bug reports can be filed [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Release notes
+
+If upgrading from Scala.js 0.6.x, make sure to read [the release notes of Scala.js 1.0.0]({{ BASE_PATH }}/news/2020/02/25/announcing-scalajs-1.0.0/) first, as they contain a host of important information, including breaking changes.
+
+This is a **minor** release:
+
+* It is backward binary compatible with all earlier versions in the 1.x series: libraries compiled with 1.0.x through 1.8.x can be used with 1.9.0 without change.
+* It is *not* forward binary compatible with 1.8.x: libraries compiled with 1.9.0 cannot be used with 1.8.x or earlier.
```
[the remark this answers is not in the document]
Comment (sjrd) [reply _posts/news/2022-02-13-announcing-scalajs-1.9.0.md]: Yes, good point. I have amended the formulation here.

File: _posts/news/2022-02-13-announcing-scalajs-1.9.0.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+Bug reports can be filed [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Release notes
+
+If upgrading from Scala.js 0.6.x, make sure to read [the release notes of Scala.js 1.0.0]({{ BASE_PATH }}/news/2020/02/25/announcing-scalajs-1.0.0/) first, as they contain a host of important information, including breaking changes.
+
+This is a **minor** release:
+
+* It is backward binary compatible with all earlier versions in the 1.x series: libraries compiled with 1.0.x through 1.8.x can be used with 1.9.0 without change.
+* Despite being a minor release, v1.9.0 is forward binary compatible with 1.8.x. It is *not* forward binary compatible with 1.7.x. Libraries compiled with 1.9.0 can be used with 1.8.x but not with 1.7.x or earlier.
```
Comment (gzm0) [inline _posts/news/2022-02-13-announcing-scalajs-1.9.0.md:37]: Nit: 1.9.0 instead of v1.9.0?

**Review by gzm0 — APPROVED** [review]
One nit: The usage of v prefix for versions feels inconsistent (see example comment).

Not sure it is super important. Feel free to merge unchanged.

**Pull request merged**